spec/views/articles/index.html.erb_spec.rb

```diff
@@ -0,0 +1,50 @@
+describe 'articles/index.html.erb', type: :view do
+
+  describe 'GET #index' do
+
+    context 'if there is one article' do
+      it 'displays the article' do
+        assign(:articles, [build_stubbed(:article, title: "Test", content: "Lorem Ipsum.")])
+
+        render
+
+        expect(rendered).to have_content("Test")
+        expect(rendered).to have_content("Lorem Ipsum.")
+      end
+    end
+
+    context 'if there is more than one article' do
+      it 'displays a list of articles' do
+        assign(:articles, [build_stubbed(:article, title: "Test1", content: "1. Lorem Ipsum."),
+                           build_stubbed(:article, title: "Test2", content: "2. Lorem Ipsum.")])
```

Comment:
Why are you using build_stubbed rather than create here and elsewhere?

Reply:
I'm using build stubbed as I don't need the articles to save to the database for this test, I just need the objects to be available. It's a performance optimisation essentially that James assisted me with.

Reply:
Yeah good choice, was just wondering if you knew the difference between that and 'create' 👍